### PR TITLE
feat!: add `import-order` plugin and rule

### PR DESCRIPTION
**Pull request by goshander**

### CODE REVIEW

File: base.js [hunk 1]
```diff
@@ -99,6 +100,33 @@ module.exports = {
...
+                    'builtin',
+                    'external',
+                    'internal',
+                    'parent',
+                    'sibling',
+                    'index'
+                ],
+                'pathGroups': [
+                    {
+                        pattern: 'shared/**',
```
Comment (ogonkov) [inline base.js]: looks like something project-specific

Comment (amje) [reply base.js]: Agreed, there should not be any specific paths

Comment (goshander) [reply base.js]: ok, removed

**Review by amje — APPROVED** [review]

File: react.js [hunk 2]
```diff
@@ -14,6 +14,58 @@ module.exports = {
...
+                    'index'
+                ],
+                'pathGroups': [
+                    {
+                        pattern: 'react',
+                        group: 'external',
+                        position: 'before'
+                    },
+                    {
+                        pattern: '*.{svg,png,jpg,jpeg,json}',
```
Comment (ogonkov) [inline react.js:41]: webp?

**Pull request merged**